File: RobotLib/src/main/java/com/acmerobotics/library/cameraoverlay/message/PingCommand.java
package com.acmerobotics.library.cameraoverlay.message;

public class PingCommand extends Command {
    public PingCommand() {
        super(MessageType.PING);
    }

    @Override
    public Class<? extends Response> getResponseClass() {
        return PongResponse.class;
    }

    @Override
    public byte[] payloadToByteArray() {
        return new byte[0];
    }

    @Override
    public void payloadFromByteArray(byte[] bytes) {

    }
}

File: CameraOverlay/src/main/java/com/acmerobotics/cameraoverlay/message/PingCommand.kt
package com.acmerobotics.cameraoverlay.message

class PingCommand: Command<Unit>(MessageType.PING) {
    override fun payloadToByteArray(): ByteArray = ByteArray(0)

    override fun payloadFromByteArray(byteArray: ByteArray) {}

    override fun getResponseType(): Class<out Response<Unit>> {
        return PongResponse::class.java
    }

}